docs/resources/service_v1.md

```diff
@@ -236,14 +236,14 @@ $ terraform import fastly_service_v1.demo xxxxxxxxxxxxxxxxxxxx@2
 
 ### Required
 
-- **backend** (Block Set, Min: 1) (see [below for nested schema](#nestedblock--backend))
 - **domain** (Block Set, Min: 1) A set of Domain names to serve as entry points for your Service (see [below for nested schema](#nestedblock--domain))
 - **name** (String) The unique name for the Service to create
 
 ### Optional
 
 - **acl** (Block Set) (see [below for nested schema](#nestedblock--acl))
 - **activate** (Boolean) Conditionally prevents the Service from being activated. The apply step will continue to create a new draft version but will not activate it if this is set to `false`. Default `true`
+- **backend** (Block Set) (see [below for nested schema](#nestedblock--backend))
```

Comment:
just curious, why backend is optional now? what would be the use case when a service does not have no backends?

Reply:
Well, beacon termination is a good example.